Lombiq.Tests.UI/Services/TeamCityMetadataReporter.cs

```diff
@@ -15,32 +16,35 @@ namespace Lombiq.Tests.UI.Services
     /// </remarks>
     public static class TeamCityMetadataReporter
     {
-        public static void ReportInt(string testName, string name, int number) =>
-            ReportNumber(testName, name, number.ToTechnicalString());
+        public static void ReportInt(UITestManifest uITestManifest, string name, int number) =>
```

Comment:
A `uITestManifest` helyett jobb lenne `uiTestManifest` szerintem olvashatóság érdekében (is), mert a betűszavakat a casing szempontjából egy szónak vesszük. Pl. ha egy változó nevében benne van az, hogy API, akkor abból a változónév elején általában `api` lesz (nem tudom hol láttam ilyet a kódunkban, de `apiUser` ugrik be elsőre) és nem `aPI`, azon kívül pedig `Api`.

Reply:
Jaja, ez csak egy typo volt (de legalább következetes :D).

Reply:
Hát ja, legalább EZ a search and replace!